### PR TITLE
added "insecure" challenge

### PR DESCRIPTION
**Pull request by shaymor**
hi yusuf! accept this pls 

Problem Type: General

### CODE REVIEW

File: insecure/provided.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Insecure
+By Shay Mor
+
+User will use netcat to run the python file.
+
+Hints:
+1. Insecure passwords aren't very secure.
+2. in·se·cure /ˌinsəˈkyo͝or/ - able to be broken into or illicitly accessed.
+3. Just try a bunch of the most common passwords lol.
```
Comment (kna27) [inline insecure/provided.txt:9]: "lol" isn't professional

**Review by kna27 — APPROVED** [review]
LGTM

File: insecure/provided.txt (hunk 1 above)
Comment (shaymor) [reply insecure/provided.txt:9]: ok lol

Comment (kna27) [reply insecure/provided.txt:9]: "lmao"

**Pull request merged**